runtime/reference/env_variables.md

```diff
@@ -112,6 +112,7 @@ The Deno runtime has these special environment variables.
 | DENO_AUTH_TOKENS     | A semi-colon separated list of bearer tokens and hostnames to use when fetching remote modules from private repositories<br />(e.g. `abcde12345@deno.land;54321edcba@github.com`) |
 | DENO_TLS_CA_STORE    | Comma-separated list of order dependent certificate stores.<br />Possible values: `system`, `mozilla`. Defaults to `mozilla`.                                                     |
 | DENO_CERT            | Load certificate authority from PEM encoded file                                                                                                                                  |
+| DENO_COVERAGE_DIR    | Set the directory for collecting coverage profile data. This option only works for [`deno test` subcommand](/runtime/reference/cli/test/).                                        |
```

Comment:
We should look into supporting it in `deno run` too, but that's a separate thing

Reply:
Agree with that idea.

related:
- https://github.com/denoland/deno/pull/28291#issuecomment-2800796412
- https://github.com/denoland/deno/issues/16440